mkosi/qemu.py

```diff
@@ -1036,7 +1036,7 @@ def add_virtiofs_mount(
 
         if want_scratch(config):
             scratch = stack.enter_context(generate_scratch_fs(config))
-            cache = "cache.writeback=on,cache.direct=on,cache.no-flush=yes"
+            cache = "cache.writeback=on,cache.direct=on,cache.no-flush=yes,aio=io_uring"
```

Comment:
does this require a particular kernel/qemu combination?

Reply:
It's been supported since qemu 5.0

Reply:
cool, didn't realize it was that old